app/services/api/webflow-helpers.ts

```diff
@@ -166,6 +180,7 @@ const convertScheduleToCardProps = (schedule: ScheduledEvent): ScheduleCardProps
         subheading: schedule["recurring-event"]?.["event-description"],
         sources: schedule["speaker-3"] ? [schedule["speaker-3"]["speaker-photo"].url] : [],
         id: schedule._id,
+        isSecondaryCallout: schedule["sponsor-is-for-a-callout"],
```

Comment:
This is one of those things where we could use types for the recurring event. For example, we could have a `RecurringEvent` and a `SponsoredRecurringEvent`. Then we'd know whether it was clickable or not. That would give us the information we need so we wouldn't need the boolean fields.

Possible ♻️ (refactor) for another PR.

Reply:
Totally agreed here. We can add this to the list so we don't use webflow next time lol